facebook-commerce-pixel-event.php

```diff
@@ -275,11 +275,11 @@ private function pixel_init_code() {
     $params = array(
       'agent' => $agent_string);
 
-    return sprintf(
+    return apply_filter('facebook_woocommerce_pixel_init', sprintf(
```

Comment:
Thanks for the pull request, but we found a small problem : this should be `apply_filters` instead of `apply_filter`. If you make this change we can merge it.

Reply:
Nevermind, It's a small change so I fixed it inline.